components/cdc/src/endpoint.rs

```diff
@@ -1146,7 +1146,13 @@ impl<T: 'static + RaftStoreRouter<E>, E: KvEngine> Endpoint<T, E> {
         let fut = async move {
             let _ = timeout.compat().await;
             // Ignore get tso errors since we will retry every `min_ts_interval`.
-            let min_ts_pd = pd_client.get_tso().await.unwrap_or_default();
+            let min_ts_pd = match causal_ts_provider {
+                // TiKV API v2 is enabled when causal_ts_provider is Some.
+                // In this scenario, get TSO from causal_ts_provider to make sure that
+                // RawKV write requests will get larger TSO after this point.
+                Some(provider) => provider.get_ts().unwrap_or_default(),
```

Comment:
Could you add some comments about the correctness of rawkv "resolved_ts"?

Reply:
Added， PTAL~